nautobot/ipam/templates/ipam/aggregate_list.html

```diff
@@ -1,14 +0,0 @@
-{% extends 'generic/object_list.html' %}
-{% load humanize %}
-
-{% block sidebar %}
-    <div class="panel panel-default">
-        <div class="panel-heading">
-            <strong><i class="mdi mdi-chart-areaspline"></i> Statistics</strong>
-        </div>
-        <ul class="list-group">
-            <li class="list-group-item">Total IPv4 IPs <span class="badge">{{ ipv4_total|intcomma }}</span></li>
-            <li class="list-group-item">Total IPv6 /64s <span class="badge">{{ ipv6_total|intcomma }}</span></li>
```

Comment:
Should we consider (if even possible before the other IPAM model changes) having something like this added to the Prefix list view?